awx_collection/plugins/doc_fragments/auth.py

```diff
@@ -32,6 +32,12 @@ class ModuleDocFragment(object):
     - If value not set, will try environment variable C(CONTROLLER_PASSWORD) and then config files
     type: str
     aliases: [ tower_password ]
+  controller_force_basic_auth:
```

Comment:
like the CONTROLLER_USERNAME and CONTROLLER_PASSWORD we might want to be able to use this as a environment variable

Reply:
it is the case already:
* https://github.com/ansible/awx/blob/6e0cff669d53688b6122d9b67d2ef85508bda137/awx_collection/plugins/module_utils/controller_api.py#L67

Maybe the description could be extended.